### PR TITLE
Add todo app example

### PR DESCRIPTION
**Pull request by IzumiSy**
Fabrix needs examples. This is an initial one: TODO app. 

We could add more examples later on (e.g. starwars, ...), but I believe TODO app is popular example so here is a good start.

## Screenshot

image

### Note: why `examples` directory in fabrix repo?

The reason why I add `examples` directory in fabrix monorepo, not in another separated repo, is productivity purpose. The whole build process is managed by turborepo, so we can quickly iterate making changes and debug in fabrix with the behaviour affected in apps in `example` directory without any external tools like [yalc](https://github.com/wclr/yalc) nor releasing packages for preview on npm.

### CODE REVIEW

File: examples/todoapp/src/columns.css
Comment (IzumiSy) [inline examples/todoapp/src/columns.css:1]: I already noticed that this stylesheet should be built-in in fabrix, but let me work on it in another PR.

File: README.md [hunk 1]
```diff
@@ -40,7 +40,15 @@ Fabrix solves this problem by automating the rendering of React components from
 >
 > We are still actively developing the features. Tell us your idea or feedback on [discussions](https://github.com/fabrix-framework/fabrix/discussions).
 
-See the instruction at [the quick start](./packages/fabrix/README.md#quick-start) to use fabrix in your React app, or [online interactive editor](https://fabrix-framework.github.io/editor/) is available to see how fabrix works out without any setups.
```
Comment (IzumiSy) [inline README.md:43]: Update here to have two sections: editor and examples. 

Added a screenshot for fabrix editor in addition.

image

Comment (IzumiSy) [reply README.md:43]: Not now, but I will add a link on this image to the YouTube video later.

File: examples/todoapp/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+              @fabrixForm(
+                input: [
+                  { field: "hasDone", config: { hidden: true } }
+                  { field: "name", config: { gridCol: 9 } }
+                  { field: "priority", config: { gridCol: 3 } }
+                ]
+              ) {
+              id
+            }
+          }
```
Comment (takumiyoshikawa) [inline examples/todoapp/src/App.tsx:29]: [ask]
After pressing the button generated from this mutation, the task list does not change, and a React warning is displayed in the browser. Is this the expected behavior?

Comment (IzumiSy) [reply examples/todoapp/src/App.tsx:29]: Let me check that on my side...

Comment (IzumiSy) [reply examples/todoapp/src/App.tsx:29]: @takumiyoshikawa 
This is a bug that I opened at https://github.com/fabrix-framework/fabrix/issues/9.

I know this is pretty critical, but can I merge this PR at first? Merging this PR really help me debug the behaviour to tackle on the issue.

**Review by takumiyoshikawa — APPROVED** [review]

**Pull request merged**